### PR TITLE
added webpack hashing to build

### PR DESCRIPTION
**Pull request by neilff**

### CODE REVIEW

**Comment by neilff** [comment]
@andrewdamelio fixed the merge conflicts

**Comment by andrewdamelio** [comment]
LGTM :+1:

File: server.js [hunk 1]
```diff
@@ -22,10 +22,8 @@ if (process.env.NODE_ENV !== 'production') {
   app.use(require('webpack-hot-middleware')(compiler));
 }
 
-app.use('/dist', express.static('dist'));
-
 app.get('/', (req, res) => {
-  res.sendFile(path.join(__dirname, 'index.html'));
+  res.sendFile(path.join(__dirname, '/dist/index.html'));
```
Comment (igor-ka) [inline server.js]: if we move index.html into the src folder we wont need this, and everything can just be served from dist folder

**Comment by SethDavenport** [comment]
Don't know if this is helpful - in the past I've followed an approach where webpack actually generates index.html - https://github.com/SethDavenport/visualizations/blob/master/webpack.config.js#L20

**Comment by neilff** [comment]
This PR has the `HtmlWebpackPlugin`, the one located in `/` is used as the template and then it will generate one based on the template which will be placed in `/dist` after a build

**Comment by SethDavenport** [comment]
Oops my bad.  :shipit:

**Pull request merged**